src/app/modules/gb-analysis-module/fractalis-components/gb-fractalis-chart/gb-fractalis-chart.component.html

```diff
@@ -1,4 +1,4 @@
-<div [id]="chart.id">
+<div [id]="chart.id" style="width: 20vw; height: 20vw;">
```

Comment:
would css work here? maybe use a css class inside gb-fractalis-chart.component.css

Reply:
any specific reasons to use 'vw'? should the size of a chart relate to viewport?

Reply:
This is only temporal. This PR does not have the final solution implemented yet, only TMT-519 & TMT-559. There is a ticket to add a flexible chart size ([TMT-628](https://jira.thehyve.nl/browse/TMT-628)), so this will be changed there.
Regarding 'vw', you're right, it does not have to be like this, it was just based on the fractalis demo page, but will be changed.